src/main/java/org/junit/Assert.java

```diff
@@ -591,33 +592,21 @@ static public void assertEquals(String message, float expected,
      * <code>actual</code> for which both numbers are still
      * considered equal.
      */
-    static public void assertNotEquals(String message, float unexpected,
+    public static void assertNotEquals(String message, float unexpected,
             float actual, float delta) {
         if (!floatIsDifferent(unexpected, actual, delta)) {
-            failEquals(message, Float.valueOf(actual));
+            failEquals(message, actual);
         }
     }
 
-    static private boolean doubleIsDifferent(double d1, double d2, double delta) {
```

Comment:
These implementations (floatIsDifferent, doubleIsDifferent) were deliberate stylistic choices for readability.  I don't feel strongly about the style, but feel that muddying the git history for a different stylistic change doesn't seem worth it.